daemon/container_unix.go

```
@@ -1118,12 +1130,17 @@ func (container *Container) unmountVolumes(forceSyscall bool) error {
 func (container *Container) networkMounts() []execdriver.Mount {
 	var mounts []execdriver.Mount
 	shared := container.hostConfig.NetworkMode.IsContainer()
+	rootUID, rootGID := container.daemon.GetRemappedUIDGID()
 	if container.ResolvConfPath != "" {
 		label.Relabel(container.ResolvConfPath, container.MountLabel, shared)
 		writable := !container.hostConfig.ReadonlyRootfs
 		if m, exists := container.MountPoints["/etc/resolv.conf"]; exists {
 			writable = m.RW
 		}
+		// if we are going to mount a resolv.conf from container metadata
+		// we need to verify it is owned properly to support a daemon
+		// with user namespaces enabled
+		os.Chown(container.ResolvConfPath, rootUID, rootGID)
```

is it safe to ignore errors here